source/Outfit.cpp

```diff
@@ -43,12 +43,20 @@ const vector<string> Outfit::CATEGORIES = {
 
 void Outfit::Load(const DataNode &node)
 {
+	// Note: an outfit with no name wouldn't be global and therefore
+	// overriding wouldn't make sense (?).
+	bool isOverride = (node.Size() >= 3 && node.Token(2) == "override");
+	if(!name.empty() && isOverride)
+		*this = Outfit();
+	else if(isOverride)
+		node.PrintTrace("Warning: overriding a non-existent outfit:");
+	
 	if(node.Size() >= 2)
 	{
 		name = node.Token(1);
 		pluralName = name + 's';
 	}
-	
+
```

Comment:
Please avoid trimming whitespace in PRs. The indentation level of the neighboring blocks should be maintained.

Reply:
It was an accident. emacs' editorconfig support doesn't handle `trim_trailing_whitespace` completely. I'm not sure how to repair this without creating an entire new commit.

Reply:
You can't. Don't worry about the number of commits in your PR; it will be squash-merged into a single commit unless you perform a collaborative effort with multiple commit authors.